robottelo/hosts.py

```diff
@@ -1754,6 +1754,15 @@ def cli(self):
         self._cli._configured = True
         return self._cli
 
+    def install_satellite_or_capsule_package(self):
+        """Install Satellite/Capsule package.
+        Note: Make sure required repos are enabled before using this.
+        """
+        enable_module_cmd = f"dnf -y module enable {self.product_rpm_name}:el{settings.server.version.rhel_version} && "
+        if settings.server.version.rhel_version == 9:
+            enable_module_cmd = ""
+        return self.execute(f"{enable_module_cmd} dnf -y install {self.product_rpm_name}")
```

Comment:
```suggestion
        if not settings.server.version.rhel_version == 9:
            self.execute(f"dnf -y module enable {self.product_rpm_name}:el{settings.server.version.rhel_version}")
        return self.execute(f"dnf -y install {self.product_rpm_name}")
```

I would prefer this personally but if your way works also then it doesn't matter.

Reply:
resolving this and we can discuss in @Gauravtalreja1's comment